packages/common/src/tableIdToHex.ts

```diff
@@ -1,5 +1,6 @@
 import { Hex, stringToHex, concatHex } from "viem";
 
+// TODO: rename to `resourceIdToHex` or `resourceSelectorToHex` since it can be used with other resources than tables
```

Comment:
but this is specific to tables (namespace+name), not all resources follow that same naming scheme, right?

Reply:
I see now that we've sprinkled this throughout our deploy process etc.

Maybe worth renaming this but still exposing a table ID variant/wrapper that does ~the same thing for clarity when working with tables.